packages/gestalt/src/TextArea.js

```diff
@@ -73,7 +81,7 @@ const TextAreaWithForwardRef: React$AbstractComponent<
   };
 
   const handleFocus = (event: SyntheticFocusEvent<HTMLTextAreaElement>) => {
-    setFocused(false);
+    setFocused(true);
```

Comment:
what's the reasoning behind this change? just curious.

Reply:
It was wrong :) `handleFocus` should set focus to `true`. `handleBlur` is correctly setting it to `false`. I assume it was a typo